### PR TITLE
Update styles for toast notification

### PR DESCRIPTION
**Pull request by sajclarke**
Update styles for Toast notification:
1. Change green accent color to #00AA5E
2. Change font to Overpass

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -732,6 +732,18 @@ div {
   }
 }
 
+/**
+ * TOASTS
+ */
```
Comment (bshevchenko) [inline src/style.scss]: we already have TOASTER section above. please put this code there

File: src/style.scss [hunk 2]
```diff
@@ -732,6 +732,18 @@ div {
...
+
+.bx--toast-notification {
+  font-family: 'Overpass', serif !important;
```
Comment (bshevchenko) [inline src/style.scss]: use $font variable instead

**Review by bshevchenko — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 1 above)
Comment (sajclarke) [reply src/style.scss]: Fixed

File: src/style.scss (hunk 2 above)
Comment (sajclarke) [reply src/style.scss]: Fixed

File: src/style.scss [hunk 3]
```diff
@@ -732,6 +740,7 @@ div {
   }
 }
 
+
```
Comment (bshevchenko) [inline src/style.scss]: excess empty line

**Review by bshevchenko — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 3 above)
Comment (sajclarke) [reply src/style.scss]: Removed

**Pull request merged**